modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogSchemaDescriptor.java

```diff
@@ -100,6 +100,11 @@ public CatalogSystemViewDescriptor[] systemViews() {
         return systemViewsMap.get(name);
     }
 
+    /** Returns {@code true} if the schema contains doesn't contain any indexes or tables, otherwise {@code false}. */
```

Comment:
```suggestion
    /** Returns {@code true} if the schema doesn't contain any objects, otherwise {@code false}. */
```

out of curiosity: why system views are not counted here? I think, it would be fair to count all the objects...

Reply:
Fixed, thanks.